USERS.md

```diff
@@ -394,3 +395,4 @@ Currently, the following organizations are **officially** using Argo CD:
 1. [Zimpler](https://www.zimpler.com/)
 1. [ZipRecuiter](https://www.ziprecruiter.com/)
 1. [ZOZO](https://corp.zozo.com/)
+1. [Zoominfo](https://www.zoominfo.com/)
```

Comment:
Let's have a separate PR for these.

Reply:
@andrii-korotkov-verkada update `USER.md `based on PR review